docs/operate/validators/run-validators.mdx

```diff
@@ -52,28 +52,216 @@ flowchart TB
+
+### Create a local directory for your validator's signatures
+
+When running a validator locally, your validator will write its signatures to a local directory. This directory can be named whatever you like, just remember to use this directory when setting your configuration.
+
+```sh
+# Pick an informative name specific to the chain you're validating
+MY_VALIDATOR_SIGNATURES_DIRECTORY=/tmp/hyperlane-validator-signatures-ethereum
+
+# Create the directory
+mkdir $MY_VALIDATOR_SIGNATURES_DIRECTORY
+```
+
+
+## Configuration
+
+:::info
+
+Take a look at the [agent configuration](../agent-config.mdx) page and the [configuration reference](../config-reference.mdx) for a full list of configuration possibilities. **The list below is not complete,** however it should be enough to get started.
+
+:::
+
+Your validator takes arguments and environment variables as configuration. See below for both the general configurations and those that are specific to the setup instructions you followed.
+
+| Argument                                      | Description                                                                                                                                                                                                                                                                                                                                                  |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `--reorgPeriod`                               | The number of block confirmations a validator should wait before signing the `Mailbox` merkle root. Note that signing a root that is later invalidated (i.e. due to a re-org) is considered fraudulent behavior and will eventually be slashable.                                                                                              |
+| `--originChainName`                           | The name of the chain being validated (e.g. `ethereum`)                                                                                                                                                                                                                                                                                                      |
+| `--chains.[origin chain name].connection.url` | The RPC URL of the node for the chain you are validating. Note Polygon mainnet requires an RPC URL of an archive node e.g. `--chains.ethereum.connection.url` or `--chains.polygon.connection.url`                                                                                                                          |
```

Comment:
I think that instead of `--chains.[origin chain name].connection.url`, we now have `rpcUrl` / `rpcUrls`